graf3d/eve/inc/TEveVector.h

```diff
@@ -24,15 +26,23 @@ class TVector3;
 template <typename TT>
 class TEveVectorT
 {
+   /// The std::array guarantees that the data is contiguous in memory and we can implement safely Arr().
+   std::array<TT, 3> fXYZ {{0}};
 public:
-   TT fX, fY, fZ; // Components of the vector.
+   TT &fX = *&fXYZ[0], fY = *&fXYZ[1], fZ = *&fXYZ[3]; // Components of the vector.
```

Comment:
Ah, I see what you mean. I fixed the original source, passing array into TMath::Cross by using local version of the operator that uses xyz. I'll check and try to remove Arr() then, I really don;t want this POD to grow from 3 floats.